.gitignore

```diff
@@ -1,14 +1,17 @@
+# Temporary cache
+.cache/
+
+# Node.js modules
 node_modules/
-npm-debug.log
-.DS_Store
-preview/
-tmp/
-axeReports/
+
+# Test coverage
 coverage/
-public/
-package/package-lock.json
```

Comment:
**issue** I think we'd want to keep that as a safety for someone inadvertently running `npm install` inside `package` and then commiting it.

Reply:
Would you mind a different approach?

I'm aware we've got lots of other npm workspace directories now

For safety, there are likely lots of other files we've missed:
```
app/package.json
app/package-lock.json
app/npm-debug.log

docs/examples/webpack/package.json
docs/examples/webpack/package-lock.json
docs/examples/webpack/npm-debug.log

package/package.json
package/package-lock.json
package/npm-debug.json
```

Perhaps if we want to stop accidents everywhere:

```.gitignore
.cache
.tmp
*.zip
*.log

package-lock.json
!/package-lock.json
```

Reply:
Yeah, that's a neat solution for our growing number of workspaces.